pkg/sbom/document_test.go

```diff
@@ -1 +1,106 @@
 package sbom
+
```

Comment:
TGhese examples are cool but there is a problem, they are in the sbom namespace. You should declare these in the `sbom_test` package to make sure they work when importing the library. 

Anyone following the examples should be using the functions with the package name like:

```golang
	// Create a new protobom document
	document := sbom.NewDocument()
```

As they are, the examples will fail when copying+pasting `document := NewDocument()` directly.